### PR TITLE
Ls 101 legal services admin tool

### PR DESCRIPTION
**Pull request by milomia**

### CODE REVIEW

File: app/controllers/legal_services/admin/uploads_controller.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          redirect_to legal_services_admin_uploads_path
+        else
+          redirect_to :back
+        end
+      end
+
+      private
+
+      def upload_params
+        params.require(:legal_services_admin_upload).permit(:current_accredited_suppliers, :geographical_data_all_suppliers, :lot_1_and_lot_2_comparisons, :master_vendor_contacts, :neutral_vendor_contacts, :pricing_for_tool, :supplier_lookup, :current_accredited_suppliers_cache, :geographical_data_all_suppliers_cache, :lot_1_and_lot_2_comparisons_cache, :master_vendor_contacts_cache, :neutral_vendor_contacts_cache, :pricing_for_tool_cache, :supplier_lookup_cache)
```
Comment (iceraluk) [inline app/controllers/legal_services/admin/uploads_controller.rb]: your files will probably have a different name here?

File: m [hunk 2]
```diff
@@ -0,0 +1,3338 @@
+Running RuboCop...
```
Comment (iceraluk) [inline m]: remove file?

File: db/schema.rb [hunk 3]
```diff
@@ -17,15 +17,6 @@
   enable_extension "plpgsql"
   enable_extension "postgis"
 
-  create_table "facilities_management_buildings", id: false, force: :cascade do |t|
```
Comment (iceraluk) [inline db/schema.rb]: might be worth matching the schema with master (as we might be missing some migrations and we don't want to edit the schema through PRs without migrations)

Comment (milomia) [reply db/schema.rb]: schema from master

**Pull request merged**